CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 Interested in contributing? We'd love your help!
 
+If you run into problems, or find something confusing, please share it with us in [this discussion](https://github.com/pikapkg/snowpack/discussions/958). A great experience for (new) contributors is very important to us.
+
+Please note that Snowpack is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating you agree to abide by its terms.
```

Comment:
I'd make this more powerful. I like how Rome does it:

> Our Discord server is open for help and more adhoc discussion. All activity on the Discord is still moderated and will be strictly enforced under the project's Code of Conduct.

with links added to discord server, code of conduct, etc.

Reply:
sidenote: I really love Rome's contributing format, and think we could do well to follow something closer to it with sections

Reply:
I'll check it out!

Reply:
Like this?

```suggestion
Please note that all activity on the [`pikapkg/snowpack` repository](https://github.com/pikapkg/snowpack) and our [Discord](https://discord.gg/rS8SnRk) is moderated and will be strictly enforced under Snowpack's [Contributor Code of Conduct](CODE_OF_CONDUCT.md).
```

Reply:
yup, but also would love to see us add a bit about "Our Discord server is open for help and more adhoc discussion." (introducing the concept of the Discord, and what it is used for, to this doc).

Reply:
I'll do a follow up